MaterialLib/MPL/Properties/Properties.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * \file
+ * \author Norbert Grunwald
+ * \date   Sep 7, 2017
+ *
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#pragma once
+
+#include "Constant.h"
```

Comment:
This file only contains one include.

Reply:
Same as for components, the full implementation has many, and the properties include is specified once in the process.
Other properties (and also components) which are not tested in the follow-up PR, are excluded.